### PR TITLE
fix(odata-service): Add mock config to ui5-local.yaml

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/106

Adds mock server config to ui5-local.yaml.

### CODE REVIEW

File: packages/odata-service-writer/src/index.ts [hunk 1]
```diff
@@ -89,6 +89,9 @@ async function generate(basePath: string, data: OdataService, fs?: Editor): Prom
         proxyLocalMiddleware.config,
         proxyLocalMiddleware.comments
     );
+
+    const mwMock = getMockServerMiddlewareConfig(data);
+    await addMiddlewareConfig(fs, basePath, 'ui5-local.yaml', mwMock);
```
Comment (ullasholla) [inline packages/odata-service-writer/src/index.ts:94]:
```suggestion
    const mockserverMiddleware = getMockServerMiddlewareConfig(data);
    await addMiddlewareConfig(fs, basePath, 'ui5-local.yaml', mockserverMiddleware);
```
Use a more descriptive name? I can't use the suggestion to capture all the usages of the variable, but you get the picture.

Comment (ullasholla) [reply packages/odata-service-writer/src/index.ts:94]: Also, don't we do this only for v4 services?

**Review by ullasholla — CHANGES_REQUESTED** [review]
Thanks. 1 minor point about using a more descriptive variable name (`mwMock` -> `mockserverMiddleware`) and a clarification about generating this for v2 & v4 services. My understanding was that it was relevant for v4 services.

Cheers for updating the snapshots.

**Review by devinea — APPROVED** [review]

File: packages/odata-service-writer/src/index.ts (hunk 1 above)
Comment (ullasholla) [reply packages/odata-service-writer/src/index.ts]: One more clarification: don't we do this conditionally in tools-suite? I see a check for a v4 service there.

Comment (ullasholla) [reply packages/odata-service-writer/src/index.ts]: Thanks @IainSAP

**Pull request merged**